cmake/Modules/merge_static_libs.cmake

```diff
@@ -63,6 +63,10 @@ function(merge_static_libs target in_target)
         if(link_dirs)
             target_link_directories(${target} PUBLIC ${link_dirs})
         endif()
+        get_target_property(include_dirs ${lib} INTERFACE_INCLUDE_DIRECTORIES)
```

Comment:
That is for the deps which is not needed as avif doe snot publicly depend on other headers.
Please try the following outside this for loop instead.
```
get_target_property(include_dirs ${in_target} INTERFACE_INCLUDE_DIRECTORIES)
if(include_dirs)
  target_include_directories(${target} PUBLIC ${include_dirs})
endif()
```

Reply:
Thanks for the comment. Updated.